cmd/server-main.go

```diff
@@ -356,6 +360,10 @@ func serverMain(ctx *cli.Context) {
 		logger.Fatal(err, "Unable to initialize policy system")
 	}
 
+	if globalIsDiskCacheEnabled {
```

Comment:
this check is already on line 240 - duplicated

Reply:
Ah the rebase bug @poornas